lib/desktop/cli.rb

```diff
@@ -250,14 +250,19 @@ def cli_syntax(command, args_str = nil)
     end
 
     command :resize do |c|
-      cli_syntax(c, 'DESKTOP WIDTHxHEIGHT')
+      cli_syntax(c, 'DESKTOP [WIDTHxHEIGHT]')
```

Comment:
This causes a couple of issues with mandatory / prohibited arguments.  Is there anything you can do to have the below print error messages such as `insufficient arguments for command 'resize'` or `excess arguments for command 'resize'`?

```
[vagrant@chead1 flight-desktop]$ /opt/flight/bin/bundle exec ./bin/desktop resize --avail 863c5e5b 640x480
640x480
800x600
<SNIP>
[vagrant@chead1 flight-desktop]$ /opt/flight/bin/bundle exec ./bin/desktop resize 863c5e5b 
desktop: invalid geometry string
```